From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: a821 <a821@nospam.mail.de>
Date: Sat, 24 Oct 2026 09:27:31 +0200
Subject: [PATCH] fix deprecation warnings

* remove LICENSE classifiers
* remove pinned build dependencies (unneeded for arch)
---
 pyproject.toml | 2 +-
 setup.cfg      | 2 --
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 0a176368..47bbb9d0 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
     "setuptools",
-    "setuptools_scm<8.0"
+    "setuptools_scm"
 ]
 build-backend = "setuptools.build_meta"
 
diff --git a/setup.cfg b/setup.cfg
index 6ded2fac..790b22fc 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -15,8 +15,6 @@ license_file = LICENSE
 classifiers =
     Development Status :: 5 - Production/Stable
     Intended Audience :: Developers
-    License :: OSI Approved :: BSD License
-    License :: OSI Approved :: Apache Software License
     Programming Language :: Python
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3.11
-- 
2.56.0

